librz/core/rop.c

```diff
@@ -1435,7 +1500,7 @@ static bool update_end_gadget(int *i, const int ropdepth, RzRopEndListPair **end
  * filters results based on the grep argument and request mask. Outputs results to
  * the provided state object.
  */
-RZ_API RzCmdStatus rz_core_rop_search(RZ_NONNULL RzCore *core, RZ_NONNULL RZ_OWN RzRopSearchContext *context) {
+RZ_API RzCmdStatus rz_core_rop_search(RZ_NONNULL RzCore *core, RZ_NONNULL RZ_BORROW RzRopSearchContext *context) {
```

Comment:
I don't think the `RZ_BORROW` is necessary, am i wrong? @XVilka

Reply:
Right, it's unnecessary